### PR TITLE
Management of the Windows agent installation and some variables.

### PR DESCRIPTION
**Pull request by KhatibMechentel**
##### SUMMARY

Role zabbix_agent :

- Windows agent installation : Management of the uninstallation of the old version of the agent

- New variables  : Linux plugins directory, Windows  

##### ISSUE TYPE

- Feature Pull Request

##### COMPONENT NAME

- zabbix_agent role

##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

File: roles/zabbix_agent/tasks/Windows.yml [hunk 1]
```diff
@@ -1,26 +1,26 @@
 ---
 - name: "Windows | Set default architecture"
-  ansible.builtin.set_fact:
```
Comment (pyrodie18) [inline roles/zabbix_agent/tasks/Windows.yml:3]: Not sure why you moved away from fully qualified modules, but please put them back in place

File: roles/zabbix_agent/templates/zabbix_agent2.conf.j2 [hunk 2]
```diff
@@ -83,16 +83,11 @@ Timeout={{ zabbix_agent2_timeout }}
 Include={{ zabbix_agent_win_include }}
 {% else %}
 Include={{ zabbix_agent2_include }}/{{ zabbix_agent2_include_pattern }}
-{% endif %}
```
Comment (pyrodie18) [inline roles/zabbix_agent/templates/zabbix_agent2.conf.j2:86]: Is there a reason to get rid of this block?

**Review by pyrodie18 — CHANGES_REQUESTED** [review]
Please see inline comments.  Please add change fragment, and also update the documentation with the variables that you added.

**Comment by KhatibMechentel** [comment]
Hello.
I'm going to work on another feature.
I close this one.

**Pull request closed without merging**